### PR TITLE
S11 product page tabs

### PR DESCRIPTION
**Pull request by APaltrow**
Task : S11 - Product page - tabs
- Added Product tabs:  Description, Reviews and Questions;
- Added ui adaptivity and skeleton loaders;
- Added logic for tabs switching;
- Modified json and added extera api call for additional info;
-

### CODE REVIEW

File: src/types/store/productStateTypes.ts [hunk 1]
```diff
@@ -11,4 +12,5 @@ export interface ProductsState extends IState {
 
 export interface ProductState extends IState {
   product: null | IProduct;
+  additionalInfo: null | IAdditionalInfo;
```
Comment (Yuliia143) [inline src/types/store/productStateTypes.ts]: It's better to have IAdditionalInfo | null, type is first

File: src/components/Product/ProductSkeleton/TabsSkeleton.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+import skeletonStyle from './ProductSkeleton.module.scss';
+
+export const TabsSkeleton: FC = () => {
+  return (
+    <div className={tabsStyle.container}>
+      <div className={tabsStyle.tabs}>
+        <div
+          className={tabsStyle.tabs_list}
+          style={{ padding: '10px' }}
```
Comment (Yuliia143) [inline src/components/Product/ProductSkeleton/TabsSkeleton.tsx]: Don't like this inline styles

File: src/types/store/productStateTypes.ts (hunk 1 above)
Comment (APaltrow) [reply src/types/store/productStateTypes.ts]: Swaped types for null to come last.

File: src/components/Product/ProductSkeleton/TabsSkeleton.tsx (hunk 2 above)
Comment (APaltrow) [reply src/components/Product/ProductSkeleton/TabsSkeleton.tsx]: Removed inline styles.

**Review by Yuliia143 — APPROVED** [review]

**Review by vynnykpff — APPROVED** [review]

**Pull request merged**